google/cloud/bigtable/test_proxy/cbt_test_proxy.cc

```diff
@@ -0,0 +1,398 @@
+// Copyright 2022 Google LLC
```

Comment:
Here and in all other files with a copyright notice, please update the date.

Reply:
Done